theme/sass/components/_card-component.scss

```diff
@@ -124,4 +136,18 @@
       }
     }
   }
+
+  .show-more {
+    margin: 0 15px;
+    padding: 15px;
+    border-radius: 10px;
+    background-color: $tertiary-color;
+    color: $accent-color-one;
+    font-size: $font-xsmall;
+    font-weight: bold;
+
+    &:hover {
+      cursor: pointer;
+    }
+  }
```

Comment:
In other places on the site (donation button, contact form) the call to action has been orange and black on hover, please make this consistent.

This would be a good time to create a standard button class and variants, that way whenever you need to add a  button you can add the classes to it and It'll be styled correctly. Look at the buttons on this page for inspiration: https://www.uwl.ac.uk/themes/custom/rutherford/pattern-lab/public/?p=viewall-atoms-buttons

Reply:
okay thank you